README.md

```
@@ -9,9 +9,11 @@ Whether you're just getting started with dbt Core™ or looking to streamline yo
 
 ## Features
 
-#### Easy Database Connectivity
+#### Easy Database & Data Lake Connectivity
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)